app/Main.hs

```diff
@@ -0,0 +1,352 @@
+module Main (main) where
+
+import Chronos (Time (..), Timespan (..), datetimeToTime, decode_YmdHMS, now, second, w3c)
+import qualified Data.ByteString as SBS
+import qualified Data.ByteString.Char8 as BSC8
+import Data.Maybe (fromJust)
+import Data.Text (Text)
+import Data.Text.Display
+import qualified Data.Text.Encoding as TE
+import qualified Data.Text.IO as TIO
+import Data.Word
+import OTP.Commons
+import OTP.HOTP
+import OTP.TOTP
+import Options.Applicative
+import Sel (secureMain)
+import qualified Sel.HMAC.SHA256 as SHA256
+import qualified Sel.HMAC.SHA512 as SHA512
+import System.Exit (exitFailure)
+import Torsor (scale)
```

Comment:
Could you split the third-party imports in a paragraph at the top and the first-party (OTP.*) imports in a paragraph below?